### ISSUE TITLE
new bug from last commit "LayerNorm" ?

### ISSUE THREAD

**Issue by thegodone** [aimat-lab/gcnn_keras#41]
File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/itetko/anaconda3/lib/python3.8/site-packages/kgcnn-1.1.2-py3.8.egg/kgcnn/literature/AttentiveFP.py", line 7, in <module>
    from kgcnn.layers.keras import Dense, Dropout, LayerNormalization
ImportError: cannot import name 'LayerNormalization' from 'kgcnn.layers.keras' (/Users/itetko/anaconda3/lib/python3.8/site-packages/kgcnn-1.1.2-py3.8.egg/kgcnn/layers/keras.py)
any help ?

**Comment by PatReis** [comment]
Yes, sorry. I moved the layer normalization to `kgcnn.layers.norm` and renamed it to `GraphLayerNormalization`. I thought that I correctly fixed the import and renaming refactoring, but I will check again. 

The reason for this that I want to make the layer to properly support the axis argument and in the future add the GraphNormalization as suggested by https://arxiv.org/abs/2009.03294. 

But the main reason is that tensorflow version 2.7 does not allows to have custom layers with the same name anymore.
I did put them in a custom package namespace but that does not sop version 2.7 from giving at least a warning (sometimes error from direct declaration).
Maybe this is an issue with tensorflow but I want to make sure, so I sadly have to refactor and rename the layers at the moment...
